backend/src/threads/services/comment_service.py

```diff
@@ -0,0 +1,55 @@
+import uuid
+from datetime import datetime
+
+from fastapi import HTTPException, Request
+from sqlalchemy.orm import Session
+
+from src.auth.utils import get_user_id_from_request
+from src.threads.repository.comment_repository import CommentRepository
+from src.threads.schemas.model_schema import (
+    CommentCreate,
+    CommentSchema,
+    CommentUpdate,
+)
+
+
+class CommentService:
+    def __init__(self, repository: CommentRepository):
+        self.repository = repository
+
+    def comment_thread(
+        self, request: Request, comment: CommentCreate, db: Session
+    ) -> bool:
+        user_id = get_user_id_from_request(request)
+        if not user_id:
+            raise HTTPException(
+                status_code=401, detail="Cannot find user_id in token payload."
+            )
+        new_comment = CommentSchema(
+            review_id=comment.review_id,
+            user_id=user_id,
+            content=comment.content,
+            created_at=datetime.utcnow(),
+        )
+        return self.repository.add_comment_to_review(new_comment, db)
```

Comment:
maybe we should return atleast 200 ? if we doing it its okay

Reply:
ok